internal/cli/filtering.go

```diff
@@ -117,10 +116,12 @@ func (f Filtering) ExpandNamedFilters(filters map[string]string, expandedFilters
 				f.ModifiedAfter = parsedFilter.ModifiedAfter
 			}
 
-			if f.Match == "" {
+			if len(f.Match) == 0 {
 				f.Match = parsedFilter.Match
-			} else if parsedFilter.Match != "" {
-				f.Match = fmt.Sprintf("(%s) AND (%s)", f.Match, parsedFilter.Match)
+			} else if 0 < len(parsedFilter.Match) {
+				for _, pmatch := range parsedFilter.Match {
+					f.Match = append(f.Match, pmatch)
+				}
```

Comment:
Now that we use slices, this can all be simplified to:

```go
f.Match = append(f.Match, parsedFilter.Match...)
```

`append` is a variadic function and [in Go you can use the `...` spread operator](https://go.dev/ref/spec#Passing_arguments_to_..._parameters) to pass each element of a slice to a variadic function.